templates/nginx_cfg.tmpl

```diff
@@ -28,6 +29,10 @@ server {{
         {NGINX_CACHE_ALL};
     }}
 
+    location = /stub_status {{
+      stub_status;
+    }}
+
     access_log /dev/stdout content_cache;
     error_log /dev/stdout info;
     access_log /var/log/nginx/access.log content_cache;
```

Comment:
A bit out of scope maybe but I'm thinking that we should specify the log format of nginx since we rely on the IP being first. The default log format could change one day

Reply:
In fact it is specified, go to files/nginx-logging-format.conf in the project directory :)